src/phoenix/trace/tracer.py

```diff
@@ -17,26 +19,39 @@ def export(self, _: Any) -> None:
 
 
 _DEFUNCT_MSG = (
-    "DEFUNCT: `Tracer` is a defunct class in the current version of Phoenix. "
-    "It no longer has any purpose or functionality and will be removed in the future."
+    "DEFUNCT: `Tracer` is now a defunct class in the current version of Phoenix and is being "
+    "deprecated. It no longer has any purpose or functionality and will be removed in the future."
 )
-_USE_ENV_MSG = (
-    "Setting endpoint through the HttpExporter is no longer supported. "
-    'Use environment variables instead, e.g. os.environ["PHOENIX_PORT"] = "54321"'
+
+_USE_ENV_MSG = """
+Setting the Phoenix endpoint through the HttpExporter is no longer supported.
+Please use environment variables instead:
+  - os.environ["PHOENIX_HOST"] = "127.0.0.1"
+  - os.environ["PHOENIX_PORT"] = "54321"
+  - os.environ["PHOENIX_COLLECTOR_ENDPOINT"] = "http://127.0.0.1:54321"
+"""
+
+_ON_APPEND_DEPRECATION_MSG = (
+    "OpenInference has been updated for full OpenTelemetry compliance. The ability to set "
+    "`on_append` callbacks are removed."
 )
 
 
 class Tracer:
     _exporter: Any
 
-    def __init__(self, *args: Any, **kwargs: Any) -> None:
-        for arg in args:
-            if isinstance(arg, HttpExporter):
-                logger.warning(_USE_ENV_MSG)
-        if "exporter" in kwargs:
+    def __init__(
+        self,
+        exporter: Any = None,
+        on_append: Optional[Callable[[List[Span]], None]] = None,
```

Comment:
suggestion: let's start eliminating the dependencies
```suggestion
        on_append: Optional[Callable[[List[Any]], None]] = None,
```